modules/nw-ovn-kubernetes-live-migration-about.adoc

```diff
@@ -0,0 +1,88 @@
+== Considerations for live migration to the OVN-Kubernetes network plugin
+
+Cluster administrators should consider the following before using the live migration method to the OVN-Kubernetes network plugin:
+
+* The live migration procedure is unsupported for clusters with OpenShift SDN multitenant mode enabled. 
+
+* Egress router pods block the live migration process. They must be removed before beginning the live migration process.
+
+* During the live migration, multicast, egress IP addresses, and egress firewalls are temporarily disabled. They can be migrated from OpenShift SDN to OVN-Kubernetes after the live migration process has finished.
+
+* Although a rollback procedure is provided, the migration is intended to be a one-way process. For users that want to rollback to OpenShift-SDN, migration from OpenShift-SDN to OVN-Kubernetes must have succeeded. Users can follow the same procedure below to migrate to the OpenShift SDN network plugin from the OVN-Kubernetes network plugin.
+
+* The live migration not supported on HyperShift clusters.
+
+* OpenShift SDN does not support IPsec. After migration, cluster administrators can enable IPsec.
+
+* OpenShift SDN does not support IPv6. Consequently, live migration to OVN-Kubernetes with IPv6 is not supported. Cluster administrators can enable dual-stack if necessary after migration is completed.
+
+* The cluster MTU is the MTU value for pod interfaces. It is always less than your hardware MTU to account for the cluster network overlay overhead. The overhead is 100 bytes for OVN-Kubernetes and 50 bytes for OpenShift SDN.
++
+During the live migration, both OVN-Kubernetes and OpenShift SDN run in parallel. OVN-Kubernetes manages the cluster network of some nodes, while OpenShift SDN manages the cluster network of others. To ensure that cross-CNI traffic remains functional, the Cluster Network Operator updates the routable MTU to ensure that both CNIs share the same overlay MTU. As a result, after the migration has completed, the cluster MTU is 50 bytes less.
+
+* Some parameters of OVN-Kubernetes cannot be changed after installation. The following parameters can be set before starting the live migration:
+
+** `InternalTransitSwitchSubnet`
+** `internalJoinSubnet`
+
+* Unless otherwise configured, OVN-Kubernetes uses the following IP address ranges:
+** `100.64.0.0/1`. This IP address range is used for `internalJoinSubnet` of OVN-Kubernetes by default. If this IP address range is already in use, enter the following command to update it to `100.63.0.0/16`:
++
+[source,terminal]
+----
+$ oc patch network.operator.openshift.io cluster --type='merge' -p='{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"ipv4":{"internalJoinSubnet": "100.63.0.0/16"}}}}}'
+----
+** `100.88.0.0/16`. This IP address range is used for `internalTransSwitchSubnet` of OVN-Kubernetes by default. If this IP address range is already in use, enter the following command to update it to `100.99.0.0/16`:
```

Comment:
Suggest update it to be` If this IP address range is already in use in OpenShift SDN`

Reply:
OVN-K shall not use this subnet if it's already used by any of the customer's networks, not just by OpenShiftSDN.